ocrd_eval.schema.yml

```diff
@@ -0,0 +1,192 @@
+        description: The workspace containing the GT
+
+      workflow_steps:
+        type: object
+        description: Human readable description of the individual steps in the workflow (for UI)
+        patternProperties:
+          '^[0-9]+$':
+            type: string
+            description: Description of this workflow step
+
+      workflow_model:
+        type: string
+        description: Human readable name of the main model used for recognition in the OCR workflow (for UI)
+
+      eval_tool:
+        type: string
+        description: Human readable name and version of evaluation tool used (for UI
+
+      document_metadata:
+        type: object
+        title: Bibliographical and typographical metadata about the work to be evaluated
+        properties:
+
+          publication_year:
+            type: number
+            description: Year he document was originally published
+
+          publication_century:
+            type: string
+            description: Century he document was originally published
+            pattern: '[12][0-9]{3}-[12][0-9]{3}'
+
+          publication_decade:
+            type: string
+            description: Decade the document was originally published
+            pattern: '[12][0-9]{3}-[12][0-9]{3}'
+
+          number_of_pages:
+            type: number
+            description: Number of pages in this work (i.e. the number of images in the gt_workspace)
+
+          layout:
+            type: string
+            enum: ['simple', 'complex']
+
+          fonts:
+            type: array
+            items:
+              type: string
+              enum: ['antiqua', 'fraktur']
+
+      provenance:
+        type: object
+        description: Information on which tools in which version were used in determining metrics
+        properties:
+          paramters:
+            type: object
+            description: Parameters passed to the evaluation processor
+
+  EvaluationReport:
+    type: object
+    additionalProperties: false
+    description: The metrics measured for this document
+    properties:
+      document_wide:
+        type: object
+        description: Document-wide metrics
+        #properties: { $ref: '#$defs/EvaluationMetrics' }
+      by_page:
+        type: array
+        description: Metrics page-by-page
+        items:
+          type: object
+          allOf:
+            - properties:
+                page_id:
+                  type: string
+                  description: PAGE ID
+            #- properties: { $ref: '#$defs/EvaluationMetrics' }
+
+  EvaluationMetrics:
+
+    cer:
+      description: CER calculated over the text of a whole page (in by_page) or combined text of all pages (in document_wide)
+
+    cer_mean:
+      description: Arithmetic mean of the page-wise CER (in document_wide) or regions on a page (in by_page)
+
+    cer_median:
+      description: Median of the page-wise CER (in document_wide) or regions on a page (in by_page)
```

Comment:
We haven't explained these in the definitions. I'd prefer to have only defined metrics in the schema (i.e. we either add them to the defs or leave them be for the moment).

Reply:
Comment out for now?